pkg/trace/api/api.go

```diff
@@ -392,10 +387,16 @@ func (r *HTTPReceiver) loop() {
 				// We expose the stats accumulated to expvar
 				info.UpdateReceiverStats(accStats)
 
-				for _, logStr := range accStats.Strings() {
+				infoStrings, warnStrings := accStats.Strings()
```

Comment:
It's saner to just call `accStats.Log()` here and that's all. All this logic seems unnecessary. Then, `accStats.Log()` can take care of everything.

Reply:
Done